ThunderCloud/ImageListItem.swift

```diff
@@ -11,58 +11,59 @@ import ThunderTable
 
 /// A subclass of `ListItem` which displays an image in the table row at it's original aspect ratio
 open class ImageListItem: ListItem {
-	
-	override open var cellClass: UITableViewCell.Type? {
-		return TableImageViewCell.self
-	}
-	
-	override open var image: UIImage? {
-		get {
-			if super.image == nil {
-				
-				let bundle = Bundle(for: ImageListItem.self)
-				let transparentImage = UIImage(named: "transparent", in: bundle, compatibleWith: nil)
-				return transparentImage?.resizableImage(withCapInsets: .zero, resizingMode: .tile)
-			}
-			
-			return super.image
-		}
-		set {
-			super.image = newValue
-		}
-	}
-	
-	override open var estimatedHeight: CGFloat? {
-		return imageHeight(constrainedTo: UIScreen.main.bounds.width)
-	}
-	
-	open func imageHeight(constrainedTo width: CGFloat) -> CGFloat? {
-		guard let image = image else { return nil }
-		let aspectRatio = image.size.height / image.size.width
-		return aspectRatio * width
-	}
-	
-	override open func configure(cell: UITableViewCell, at indexPath: IndexPath, in tableViewController: TableViewController) {
-		
-		super.configure(cell: cell, at: indexPath, in: tableViewController)
-		
-		guard let imageCell = cell as? TableImageViewCell else { return }
-		imageCell.cellImageView?.contentMode = .scaleAspectFill
-		imageCell.layer.masksToBounds = true
-		
-		if let imageHeight = imageHeight(constrainedTo: tableViewController.view.frame.width) {
-			imageCell.imageHeightConstraint?.constant = imageHeight
-		}
-	}
-	
-	override open var accessoryType: UITableViewCell.AccessoryType? {
-		get {
-			return UITableViewCell.AccessoryType.none
-		}
-		set {}
-	}
-	
-	override open var selectionStyle: UITableViewCell.SelectionStyle? {
-		return UITableViewCell.SelectionStyle.none
-	}
+    
+    override open var cellClass: UITableViewCell.Type? {
+        return TableImageViewCell.self
+    }
+    
+    override open var image: UIImage? {
+        get {
+            if super.image == nil {
+                
+                let bundle = Bundle(for: ImageListItem.self)
+                let transparentImage = UIImage(named: "transparent", in: bundle, compatibleWith: nil)
+                return transparentImage?.resizableImage(withCapInsets: .zero, resizingMode: .tile)
+            }
+            
+            return super.image
+        }
+        set {
+            super.image = newValue
+        }
+    }
+    
+    override open var estimatedHeight: CGFloat? {
+        return imageHeight(constrainedTo: UIScreen.main.bounds.width)
```

Comment:
It will return the incorrect `estimatedHeight` but unfortunately the way `ThunderTable` is written we currently can't go off anything else. Fortunately all the `estimagedHeight` is used for is sizing the scroll indicator in the table view so it's not a mahoosive worry to me really :D